utils/coreutils/utils.go

```diff
@@ -516,3 +527,21 @@ func ValidateMinimumVersion(product MinVersionProduct, currentVersion, minimumVe
 	}
 	return nil
 }
+
+func SplitRepoAndServerId(remoteEnv string) (serverID string, repoName string, err error) {
+	serverAndRepo := os.Getenv(remoteEnv)
+	if serverAndRepo == "" {
+		log.Debug(remoteEnv, "is not set")
+		return "", "", nil
+	}
+	// The serverAndRepo is in form of '<ServerID>/<RemoteRepo>'
+	lastSlashIndex := strings.LastIndex(serverAndRepo, "/")
```

Comment:
Please consider using `strings.Split(serverAndRepo, "/")` instead.
This may simplify the below if statement.